src/libraries/System.Security.Cryptography/tests/AesGcmTests.cs

```diff
@@ -23,7 +25,7 @@ public static void EncryptTamperAADDecrypt(int dataLength, int additionalDataLen
             byte[] ciphertext = new byte[dataLength];
             byte[] key = new byte[16];
             byte[] nonce = new byte[AesGcm.NonceByteSizes.MinSize];
-            byte[] tag = new byte[AesGcm.TagByteSizes.MinSize];
+            byte[] tag = new byte[AesGcm.TagByteSizes.MaxSize];
```

Comment:
Do we still need to make this/these edits now that the value has been updated for Apple OSes?

Reply:
Nope. Thanks.